.github/workflows/test_frontend_e2e_live_prod.yml

```diff
@@ -0,0 +1,89 @@
+# Test Frontend deployment using Playwright End-to-End tests.
+# todo: 1. run local: (i) start backend (a) local backend in GH action, or (b) use dev's backend URL (won't always work), (ii) set to on:pr&push
+# todo: 2. after '1': expose params to workflow_dispatch: so can choose which env we want to test, in case we don't want to run all 3 at once
+
+name: Test, Frontend Prod, Playwright E2E
```

Comment:
## Prod
- Add: Production GH action that tests on prod using the code / playwright tests that were written at the time prod was deployed.